.docker/Dockerfile

```diff
@@ -59,9 +67,10 @@ RUN mkdir /library_pkgs && \
 FROM ros:noetic-ros-core
 ARG SCANCONTROL_SDK_VERSION
 
-RUN apt-get update && \
-    apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends\
       intltool \
+      libglib2.0-dev \
+      libusb-1.0-0-dev \
```

Comment:
This should not be needed if the `--required` tag is used correctly when creating the `.deb` file.